.github/workflows/autoprerelease.yml

```
@@ -38,3 +40,11 @@
           tag_name: ${{ steps.changelog.outputs.tag }}
           release_name: ${{ steps.changelog.outputs.tag }}
           body: ${{ steps.changelog.outputs.clean_changelog }}
+
+      - name: Push changes
+        if: ${{ steps.changelog.outputs.skipped == 'false' }}
+        run: |
```

actionlint (1.7.4)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)